spec/components/pretty_text_spec.rb

```diff
@@ -553,7 +562,7 @@ def cook(*args)
     it "strips out unicode bidirectional (bidi) override characters and replaces with a highlighted span" do
       code = <<~MD
          X
-         ```js
+         ```auto
```

Comment:
Why is a change to this test needed? Shouldn't `js` still work?

Reply:
No, before this change js was not in the list of classes (could be javascript), you can see the test was actually expecting `code class="lang-auto"` which means it wasn't expecting to recognise js (it could if it was in list of classes in site setting). But now that any unkown class is converted to a code block, js would be converted to `data-wrap="js"` I could have changed the rest of the test, but this test is about testing special unicode chars, not this, so going with auto is the simplest change here.